addons/mass_mailing_sms/wizard/mailing_sms_test.py

```diff
@@ -48,15 +62,17 @@ def action_send_sms(self):
             notification_messages.append(_('The following numbers are not correctly encoded: %s',
                 ', '.join(invalid_numbers)))
 
+        sms_uuid_to_number_map = {sms.uuid: sms.number for sms in new_sms_messages_sudo}
         for sent_sms in sent_sms_list:
+            number = sms_uuid_to_number_map.get(sent_sms.get('uuid'))
             if sent_sms.get('state') == 'success':
                 notification_messages.append(
-                    _('Test SMS successfully sent to %s', sent_sms.get('res_id')))
+                    _('Test SMS successfully sent to %s', number))
             elif sent_sms.get('state'):
                 notification_messages.append(
                     _(
                         "Test SMS could not be sent to %(destination)s: %(state)s",
-                        destination=sent_sms.get("res_id"),
+                        destination=number,
```

Comment:
Should display number not False. https://tinyurl.com/24qloyx8

Reply:
Done would you mind taking a look. :)